crypto_kem/ntruhps40961229/META.yml

```diff
@@ -23,4 +23,14 @@ auxiliary-submitters:
   - Zhenfei Zhang
 implementations:
     - name: clean
-      version: https://github.com/jschanck/ntru/tree/3227cd42 reference implementation
+      version: https://github.com/jschanck/ntru/tree/bc5e7efe reference implementation
+    - name: avx2
+      version: https://github.com/jschanck/ntru/tree/bc5e7efe avx2 implementation
+      supported_platforms:
+          - architecture: x86_64
+            operating_systems:
+                - Linux
+                - Darwin
+            required_flags:
+                - avx2
+                - bmi2
```

Comment:
Is this a mistake? I don't think the PR has an AVX2 implementation of NTRU HPS-4096-1229?

```suggestion
```